script/generate_helm.sh

```diff
@@ -38,7 +39,7 @@ if [ -n "$DELTA_CHECK" ]; then
     echo "diff detected: $DIFF"
     DIFF=$(git diff --name-only)
     echo "files different: $DIFF"
-    exit 1
+    exit 0
```

Comment:
so always succeed?

Reply:
Ops... nooo... that was me playing with the script... thanks for catching this!